src/main/java/hello/HelloWorldHttpController.java

```diff
@@ -16,6 +16,8 @@ public String greeting(@RequestParam(value="name", required=false, defaultValue=
         if (true){
             int r = 4 + 5;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Assign this magic number 4 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3AS109)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3AS109)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3AIndentationCheck)